src/app/pages/BuyPage/components/Input/index.tsx

```diff
@@ -0,0 +1,108 @@
+import cn from 'classnames';
```

Comment:
Please name it `classNames` instead of `cn`.

Reply:
fixed